docs/visual-basic/language-reference/error-messages/default-property-propertyname1-conflicts-with-default-property-propertyname2.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Default property &#39;&lt;propertyname1&gt;&#39; conflicts with default property &#39;&lt;propertyname2&gt;&#39; in &#39;&lt;classname&gt;&#39; and so should be declared &#39;Shadows&#39;"
+title: "Default property &#39;<propertyname1>&#39; conflicts with default property &#39;<propertyname2>&#39; in &#39;<classname>&#39; and so should be declared &#39;Shadows&#39;"
```

Comment:
@v-alje and what about the `&#39;`? Keep that like that?

Reply:
@mairaw single quotes have been flattened as well now.